.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+# Description
+
+Please include a summary of relevant context/issue and your changes.
+
+# Tests
+
+Please describe the tests that you ran on TPUs to verify changes.
+
+**Instruction and/or command lines to reproduce your tests:** ...
+
+**List links for your tests (use go/shortn-gen for any internal link):** ...
+
+# Checklist
+
+Before submitting this PR, please make sure:
+- [ ] I have performed a self-review of my code.
+- [ ] I have commented on my code, particularly in hard-to-understand areas.
```

Comment:
s/I have commented on my code/I have necessary comments in my code

Reply:
Sure, updated.